packages/textInput/components/TextInput.tsx

```diff
@@ -134,6 +134,7 @@ export class TextInput<P extends TextInputProps> extends React.Component<P> {
       value,
       showInputLabel,
       appearance,
+      tooltipContent,
```

Comment:
❓ I'm not sure, but could there be a way to make sure to only pass valid HTML to the `<input>`-element?

Something like: 

```
<input {...rest.filter(prop => isValidNativeInputProp(prop) />
```

But I'm not sure if something like this exists or is possible, but this would feel a lot better than to destructure all props we do not want to pass.

Reply:
We talked about it and decided to not do it for now. If that pops up a lot again, we can look into if we want to implement something like this.